app/code/AndriiShkrebtii/CMSPageModule/etc/module.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="CMSPageModule"/>
```

Comment:
@jimmywilson111 Vendor name missed! Check the code examples - https://github.com/maksymz/dv-campus-2020-2021-magento/blob/master/app/code/DVCampus/Catalog/etc/module.xml